calibration/DOSTAN/CGINS-DOSTAN-01520__20180129.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+1520,CC_calibration_date,1517184000000,sensor calibration date (milliseconds since 1970-01-01)
```

Comment:
Resolving @rtravis-whoi comment to correct cal date in FN and in csv to 2018-01-29.

Reply:
looks good.